### DISCUSSION TITLE
Question: ¿When will week 7 be available to download?

### DISCUSSION THREAD

**Discussion by PolarChunkz** [FunkinCrew/Funkin#1536] [General]
Hey a question, does anyone know when week 7 will be available to download, because for some reason newgrounds does not load the game and I also want the tankman files, does anyone know?

**Comment by MarioLuigi124** [comment]
no

**Comment by ghost** [comment]
I think it's a newgrounds exclusive, also when the loading gets stuck reload and continue from there

**Comment by kinoko87** [comment]
In some time, it was never said when but it'll eventually be added to itch.io and here.

**Comment by xiiixvv** [comment]
On twitter Ninjamuffin said that it will be released once Newgrounds becomes the second or first result when you search up "Friday Night Funkin". Also, refresh the page or clear your browser cache if NG isnt loading the game properly.

**Comment by ghost** [comment]
crazygames is somehow result 2

**Comment by ghost** [comment]
[tankman files](https://github.com/kckarnige/fnf-week-7-code)

**Comment by ghost** [comment]
but is that everything or just the sprite

**Comment by aflacc** [comment]
ninja said he will put a download and update the github when the newgrounds one is the top result when you search friday night funkin

**Comment by ghost** [comment]
Pretty sure it's first or second

**Comment by xTHECONNORx** [comment]
> [tankman files](https://github.com/kckarnige/fnf-week-7-code)

But how do you even add that to the downloadable version?
